x-pack/plugins/infra/public/components/alerting/logs/expression_editor/editor.tsx

```diff
@@ -132,7 +132,7 @@ export const Editor: React.FC<Props> = (props) => {
   const supportedFields = useMemo(() => {
     if (sourceStatus?.logIndexFields) {
       return sourceStatus.logIndexFields.filter((field) => {
-        return (field.type === 'string' || field.type === 'number') && field.searchable;
+        return field.type === 'string' || field.type === 'number';
```

Comment:
ok, I totally mixed up `supportedFields` and `groupByFields` in my head :see_no_evil: Given that these are used in the main query's `filter`, I think `searchable` is the right check here and `aggregatable` is correct for the grouping field, which is used in aggs.


```suggestion
        return (field.type === 'string' || field.type === 'number') && field.searchable;
```

Does that make sense or did I get it the wrong way around again? :thinking: Sorry for the confusion :facepalm: